data/en/mbs_0201.json

```diff
@@ -0,0 +1,478 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
```

Comment:
should probably add 201 here for clarity

Reply:
Done